medusa/providers/torrent/json/rarbg.py

```diff
@@ -101,7 +101,7 @@ def search(self, search_strings, age=0, ep_obj=None):
             else:
                 search_params['sort'] = self.sorting if self.sorting else 'seeders'
                 search_params['mode'] = 'search'
-                search_params['search_tvdb'] = self._get_tvdb_id()
+                search_params['search_tvdb'] = self._get_tvdb_id() if ep_obj else None
```

Comment:
This should not be here.  Logic to test if show exists should be in `_get_tvdb_id` as it applies to all `_get_tvdb_id` calls.  And `_get_tvdb_id` should return falsey (`None` would probably be most appropriate) if an ID isn't available.